0029-loopback-transport/0029-loopback-transport.md

```diff
@@ -0,0 +1,33 @@
+---
+title: The Loopback Transport Protocol (LT)
+draft: 1
+---
+# Loopback Transport Protocol (LT)
+
+The Loopback Transport protocol is a technique for transporting value over the Interledger. It relies on the establishment of a temporary, direct, ILPv4 link between sender and receiver (the loopback link). The flow is as follows.
+
+## Flow
+
+The following flow describes how the sender's loopback transfer module ("LT module") interacts with the application layer protocol, the receiver, and the ILPv4 network.
+
+1. The loopback connection is established out of band. For instance, the application layer protocol may allow the receiver to specify a [BTP/2.0](../0023-bilateral-transfer-protocol/0023-bilateral-transfer-protocol.md) URL for the sender to connect to.
+1. The sender's LT module is initialized with this open connection.
+1. The sender's LT module discovers the loopback address using [IL-DCP](https://github.com/interledgerjs/ilp-protocol-ildcp).
```

Comment:
the loopback address = receiver's address, right?

Could you describe somewhere what the requirements are for the receiver? What are they running? Sounds like a BTP server and an ILDCP server. Is there anything special about how they determine the loopback address for a given connection, or is that just done by the connector / multi-plugin?

Reply:
loopback address = sender's address at the receiver

sure, i can add a description of how the receiver would behave (and as discussed in our 1:1, i'll also add a receiver in the demo, forgot to do that earlier)